### ISSUE TITLE
Color themes not 508 Compliant

### ISSUE THREAD

**Issue by jamesuhl** [IgniteUI/ig-typedoc-theme#35]
## Description
Colors chosen for many web components fail the color contrast required for Section 508 accessibility. 

## Solution
In the website and web components, darken the following colors: 
Change green (#309331) to #258826. Change your light blue (#0099ff) to #0076dc. Change any gray text using either IgniteUI/igniteui-angular#777 or #808080 to at least #767676 (for white background) or IgniteUI/igniteui-angular#666 if the text is on a light gray (#ebebeb)

## Details
Consider going through all of your color themes with ANDI tool and increasing the contrast where needed. 
The ANDI Accessibility Testing Tool is found here: (https://www.ssa.gov/accessibility/andi/help/install.html ). Another useful resource is this page: https://webaim.org/resources/contrastchecker/.  It will display all the colors that fail the WCAG contrast criteria 

![Image](https://github.com/user-attachments/assets/53ab6fa0-ece7-4133-9dd3-8929efa60cfb)
